### PR TITLE
Added Resultset Table stories

### PR DESCRIPTION
**Pull request by Lisa159**
Stories will be edited in the future when the prop **showGoToPage** is added. -> #685

### CODE REVIEW

File: lib/src/resultsetTable/ResultsetTable.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+    <DxcResultsetTable columns={columns} rows={rows} itemsPerPage={2}></DxcResultsetTable>
+  </ExampleContainer>
+);
+
+export const MiddlePage = ResultsetTableMiddle.bind({});
+MiddlePage.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const nextButton = canvas.getAllByRole("button")[2];
+  await userEvent.click(nextButton);
+};
```
Comment (raquelarrojo) [inline lib/src/resultsetTable/ResultsetTable.stories.tsx:262]: This test is not necessary as this case is already tested in the button visual tests.

File: lib/src/resultsetTable/ResultsetTable.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    <DxcResultsetTable columns={columns} rows={rows} itemsPerPage={2}></DxcResultsetTable>
+  </ExampleContainer>
+);
+
+export const LastPage = ResultsetTableLast.bind({});
+LastPage.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const nextButton = canvas.getAllByRole("button")[3];
+  await userEvent.click(nextButton);
+};
```
Comment (raquelarrojo) [inline lib/src/resultsetTable/ResultsetTable.stories.tsx:276]: This test is not necessary as this case is already tested in the paginator visual tests.

File: lib/src/resultsetTable/ResultsetTable.stories.tsx (hunk 1 above)
Comment (Lisa159) [reply lib/src/resultsetTable/ResultsetTable.stories.tsx:262]: This test is added to check functionality of the next button, to see the table state in a middle page.

File: lib/src/resultsetTable/ResultsetTable.stories.tsx (hunk 2 above)
Comment (Lisa159) [reply lib/src/resultsetTable/ResultsetTable.stories.tsx:276]: This test is added to check functionality of the last button, to see the table state in the last page.

File: lib/src/resultsetTable/ResultsetTable.stories.tsx (hunk 1 above)
Comment (raquelarrojo) [reply lib/src/resultsetTable/ResultsetTable.stories.tsx:262]: Got it! 👍🏻

File: lib/src/resultsetTable/ResultsetTable.stories.tsx (hunk 2 above)
Comment (raquelarrojo) [reply lib/src/resultsetTable/ResultsetTable.stories.tsx:276]: Got it! 👍🏻

**Comment by raquelarrojo** [comment]
Waiting for scroll in table to be merged.

**Comment by raquelarrojo** [comment]
Do you know why Chromatic is getting this change in the tabs component? @Lisa159 
image

**Comment by Lisa159** [comment]
> Do you know why Chromatic is getting this change in the tabs component? @Lisa159 image

No idea why this is happening. As you can see I didn't edit files about tabs...

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**